### PR TITLE
procutil: don't send signals to nil process

### PR DESCRIPTION
**Pull request by jazzdan**
Before: 

If you `tilt up` this branch Tilt would segfault
https://github.com/windmilleng/enhance/compare/dmiller/crash-tilt?expand=1

After:
It starts up fine and doesn't segfault

### CODE REVIEW

File: internal/engine/local/execer.go [hunk 1]
```diff
@@ -141,16 +141,16 @@ func processRun(ctx context.Context, cmd model.Cmd, w io.Writer, statusCh chan s
 	c.Stderr = w
 	c.Stdout = w
 
-	errCh := make(chan error)
+	err := c.Start()
+	if err != nil {
+		return
```
Comment (landism) [inline internal/engine/local/execer.go]: We should log `err` here and have a test for it. It looks like I only tested logging exit errors.

Comment (jazzdan) [reply internal/engine/local/execer.go]: Oh yeah good call, I meant to do that.

Comment (landism) [reply internal/engine/local/execer.go]: oh, I guess splitting this out from errCh means we no longer have one handler for Start and Wait errors. blergh.

Comment (jazzdan) [reply internal/engine/local/execer.go]: Yeah, but we could abstract out a function. :)

**Pull request merged**